### ISSUE TITLE
Error running CustomServiceSample.Android

### ISSUE THREAD

**Issue by dikokob** [AndyCW/MovesenseDotNet#8]
I get the following error on I build CustomServiceSample.Android: **Unable to remove directory "obj\Debug\81\android\bin\classes". Could not find a part of the path 'AccessibilityManagerCompat_AccessibilityStateChangeListenerImplementor.class'.**

**Comment by AndyCW** [comment]
This is a Visual Studio issue. Please manually delete the /bin and /obj folders under your Android project root folder and then rebuild - you should be ok.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by dikokob** [comment]
Thanks, that worked

**Comment by meckblack** [comment]
Enter the obj\Debug\81\android\bin\classes\..... till the end and copy out the AccessibilityManagerCompat_AccessibilityStateChangeListenerImplementor.class paste it in the 
obj\Debug\81\android\bin\ path. then delete the classes folder. 
It worked for me!!!!
